client/src/components/SingleNews/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import hover_arrow from '../../assets/img/news-hover-arrow.svg'
+import { Link } from 'react-router-dom'
+
+export default function SingleNews({ singleNews }) {
+  console.log(singleNews)
+  return (
+    singleNews.map(element => (
+      <div className='single-news' key={element.id}>
```

Comment:
u render 3 items here. How then it can be single news? maybe this map should be higher?

Reply:
it renders each news item individually and the amount depends on the amount specified in the file with array

Reply:
but then it is not single news --- it is three news

Reply:
newsBlock will be fine?)

Reply:
yes

Reply:
refactored